### DISCUSSION TITLE
.NET Core

### DISCUSSION THREAD

**Discussion by spleefian** [MicaForEveryone/MicaForEveryone#185] [General]
I installed MicaForEveryone, but it asks me to install .NET Core. I click "Yes," and nothing happens.  I went online, downloaded .NET SDK 6 (which it says is a new version of .NET Core), and reboot my computer. I open MFE again and it tells me to install .NET Core again. I click yes again, still nothing happening. I am very confused, please help me if I'm installing the wrong .NET version or anything like that.

**Comment by BORGERone** [comment]
I have the same problem. I download .NET 6.0, .NET 7.0 and .NET 7.0 x 86.
Windows 11 22H2 (OC 22621.819)

**Comment by BORGERone** [comment]
I solved the problem by installing .NET 3.1

https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-3.1.425-windows-x64-installer

**Reply by spleefian** [reply]
Thanks so much!!!

**Reply by rurumuri** [reply]
Thanks so much too!!!

**Reply by DakenRise** [reply]
nice
